group_project_v2/stage/feedback_review.py

```diff
@@ -0,0 +1,162 @@
+import itertools
+import logging
+from lazy.lazy import lazy
+from xblock.fields import String, Scope
+from xblock.validation import ValidationMessage
+from group_project_v2.stage.utils import DISPLAY_NAME_NAME, DISPLAY_NAME_HELP
+from group_project_v2.stage.base import BaseGroupActivityStage
+from group_project_v2.stage.mixins import SimpleCompletionStageMixin
+from group_project_v2.stage_components import (
+    GroupProjectTeamEvaluationDisplayXBlock, GroupProjectGradeEvaluationDisplayXBlock
+)
+from group_project_v2.utils import gettext as _, MUST_BE_OVERRIDDEN
+
+
+log = logging.getLogger(__name__)
+
+
+class FeedbackDisplayBaseStage(SimpleCompletionStageMixin, BaseGroupActivityStage):
+    NAVIGATION_LABEL = _(u'Review')
+    FEEDBACK_DISPLAY_BLOCK_CATEGORY = None
+
+    @property
+    def feedback_display_blocks(self):
+        return self.get_children_by_category(self.FEEDBACK_DISPLAY_BLOCK_CATEGORY)
+
+    @lazy
+    def required_questions(self):
+        return [
+            feedback_display.question_id for feedback_display in self.feedback_display_blocks
+            if feedback_display.question and feedback_display.question.required
+        ]
+
+    def get_reviewer_ids(self):
```

Comment:
Same question here about abstract methods, but this is a moved class rather than new. Not sure if it would be better to update all examples like this or just continue on with existing convention.